site/src/main/java/uk/nhs/digital/ps/beans/Series.java

```diff
@@ -10,4 +11,13 @@
     public String getTitle() {
         return getProperty("publicationsystem:Title");
     }
+
+    @HippoEssentialsGenerated(internalName = "publicationsystem:Summary")
+    public String getSummary() {
+        return getProperty("publicationsystem:Summary");
+    }
+
+    public HippoBean getSelfLink() {
+        return getParentBean();
+    }
```

Comment:
In an off-line discussion we agreed on adding a JavaDoc comment clarifying the function of this method.